schemas/io-package.json

```diff
@@ -0,0 +1,457 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "required": ["common"],
+    "additionalProperties": false,
+    "properties": {
+        "$schema": {
+            "description": "JSON schema for validation",
+            "type": "string"
+        },
+        "common": {
+            "required": ["name", "version", "platform", "titleLang", "news", "desc", "mode"],
+            "additionalProperties": false,
+            "description": "Contains all common instance attributes",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "Name of the adapter without ioBroker",
+                    "type": "string",
+                    "pattern": "(?i)^(?!iobroker)"
+                },
+                "version": {
+                    "description": "Current adapter version",
+                    "type": "string",
+                    "pattern": "^\\d+\\.\\d+\\.\\d+.*"
+                },
+                "platform": {
+                    "description": "Platform, normally Javascript/Node.js",
+                    "type": "string",
+                    "enum": ["Javascript/Node.js"]
+                },
+                "title": {
+                    "description": "[DEPRECATED] Longer name of adapter to show in admin",
+                    "type": "string"
+                },
+                "titleLang": {
+                    "description": "Longer name of adapter in all supported languages like {en: 'Adapter', de: 'adapter', ru: 'Драйвер'}",
+                    "type": "object",
+                    "required": ["en"],
+                    "patternProperties": {
+                        "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                            "type": "string"
+                        }
+                    },
+                    "additionalProperties": false
+                },
+                "news": {
+                    "description": "Short form of changelog in all supported languages",
+                    "type": "object",
+                    "minProperties": 1,
+                    "maxProperties": 20,
+                    "patternProperties": {
+                        "^\\d+\\.\\d+\\.\\d+.*": {
+                            "type": "object",
+                            "required": ["en"],
+                            "patternProperties": {
+                                "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                                    "type": "string"
+                                }
+                            },
+                            "additionalProperties": false
```

Comment:
I think we could extract this part (which is repeated in multiple locations) into a definition.
==> http://json-schema.org/understanding-json-schema/structuring.html#id1